include/nbl/core/execution.h

```diff
@@ -5,7 +5,7 @@
 #ifndef _NBL_CORE_EXECUTION_H_INCLUDED_
 #define _NBL_CORE_EXECUTION_H_INCLUDED_
 
-#if !defined (__clang__)
+#ifndef _NBL_BUILD_DPL_
 #include <execution>
```

Comment:
in C++20 there's now `__has_include()` which we can use to detect presence of `<execution>`